src/webui/www/public/scripts/client.js

```diff
@@ -366,6 +366,7 @@ window.addEvent('load', function () {
             document.title += " QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]";
         }else
             document.title = "qBittorrent ${VERSION} QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]";
+        $('freeSpaceOnDisk').set('html', 'QBT_TR(Free space: %1)QBT_TR[CONTEXT=StatusBar]'.replace("%1", friendlyUnit(serverState.free_space_on_disk)));
```

Comment:
`CONTEXT=StatusBar` this is not right, there are no translation in it (since #6764 isn't accepted)
you should put `CONTEXT=HttpServer` here and add an entry in "src/webui/extra_translations.h"